app/spec/services/github_checks_verifier_spec.rb

```diff
@@ -98,6 +99,17 @@
         service.fail_unless_all_success(all_checks)
```

Comment:
Please change all references in tests to the "old" `fail_unless_all_success` method.